### PR TITLE
Update usage information

### PR DESCRIPTION
**Pull request by kusumaghoshdastidar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,112 @@ Swagger configuration params that can be set in microfocus-config.js:
  - deepLinking: true
 
 For a complete list of swagger configuration params refer [Swagger configuration params](https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md#core).
+
+## Using this module in Spring Boot
+
+[Here](https://github.com/CAFapi/caf-swagger) is an example of how this swagger-ui can be used in a Spring Boot application :
```
Comment (dermot-hardy) [inline README.md]: Move the link in a level to `https://github.com/CAFapi/caf-swagger/tree/master/caf-swagger-ui`.

File: README.md [hunk 2]
```diff
@@ -25,3 +25,112 @@ Swagger configuration params that can be set in microfocus-config.js:
  - deepLinking: true
 
 For a complete list of swagger configuration params refer [Swagger configuration params](https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md#core).
+
+## Using this module in Spring Boot
+
+[Here](https://github.com/CAFapi/caf-swagger) is an example of how this swagger-ui can be used in a Spring Boot application : 
+1. Add the module as a `runtime` dependency.
```
Comment (dermot-hardy) [inline README.md:31]: Which module are you referring to now?  This one or the `com.github.cafapi:swagger-ui-annotation` one?

File: README.md [hunk 3]
```diff
@@ -25,3 +25,112 @@ Swagger configuration params that can be set in microfocus-config.js:
...
+2. Include a resource file called **microfocus-config.js** with the swagger configuration overrides, similar to the one [here](./src/main/resources/microfocus-config.js).
+Make sure to overide the "url" param to point to your swagger contract.
+3. To facilitate [serving static resources](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/servlet/config/annotation/ResourceHandlerRegistry.html) in Spring Boot override the `addResourceHandlers` function in 
+
+```java
+@Override
+public void addResourceHandlers(final ResourceHandlerRegistry registry) {
+    // configure the external-facing URI path by adding a resource handler
+    registry.addResourceHandler("/swagger/**");
+    // map that external-facing URI path internally to the physical path where the resources are actually located
```
Comment (dermot-hardy) [inline README.md]: Use proper sentence casing and add some blank lines to make this code snippet easier to read.

**Review by dermot-hardy — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (kusumaghoshdastidar) [reply README.md:31]: This one

Comment (dermot-hardy) [reply README.md:31]: Is that necessary?  I don't see it in [this](https://github.com/CAFDataProcessing/staging-service/commit/f4a051f08b522a1cf806083dd1d1b2237a88cddc) commit.

**Pull request merged**